consoleme/handlers/v2/typeahead.py

```diff
@@ -97,3 +100,58 @@ async def get(self):
             self.write(json.dumps([{"title": arn} for arn in arn_array.__root__]))
         else:
             self.write(arn_array.json())
+
+
+class SelfServiceStep1ResourceTypeahead(BaseAPIV2Handler):
+    async def get(self):
+        try:
+            type_ahead: Optional[str] = (
+                self.request.arguments.get("typeahead")[0].decode("utf-8").lower()
+            )
+        except TypeError:
+            type_ahead = None
+        if not type_ahead:
+            self.write(json.dumps([]))
+            return
+        try:
+            limit: int = self.request.arguments.get("limit")[0].decode("utf-8")
```

Comment:
nit: This type annotation will cause errors since the return type of `decode()` is `str`. Maybe make this `limit_raw: str`?